wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -804,17 +806,9 @@ def timesince_last_update(
                 return time_str
     else:
         if use_shorthand:
-            # Note: Duplicate code in timesince_simple()
-            time_period = timesince(last_update).split(",")[0]
-            if time_period == avoid_wrapping(_("0 minutes")):
```

Comment:
Would it be useful to keep this special case, but move them to when the dates are the same? Or maybe even outside that check, so it also handles the midnight case…

Reply:
I'd rather stick with the intent of the original code.